src/test/java/com/programmers/ticketparis/domain/reservation/ReservationTest.java

```diff
@@ -0,0 +1,52 @@
+package com.programmers.ticketparis.domain.reservation;
+
+import com.programmers.ticketparis.exception.ReservationException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class ReservationTest {
+
+    @Test
+    @DisplayName("예매를 생성 할 수 있다.")
+    void createReservation_Success() {
+        // given & when
+        Reservation reservation = Reservation.builder()
+                .reservationStatus(ReservationStatus.COMPLETED)
+                .scheduleId(1L)
+                .customerId(1L)
+                .build();
+
+        // then
+        assertThat(reservation).isInstanceOf(Reservation.class);
+    }
+
+    @Test
+    @DisplayName("예매 상태는 예매 완료와 예매 취소가 아닌 경우 예외를 발생시킨다.")
+    void anotherReservationStatus_Exception() {
+        // given & when & then
+        assertThatThrownBy(() -> {
+            Reservation.builder()
+                    .reservationStatus(ReservationStatus.valueOf("Proceeding"))
+                    .scheduleId(1L)
+                    .customerId(1L)
+                    .build();
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
`P2`
- 해당 테스트 코드는 아래와 같이 간소화 될 수 있지 않을까요?
```suggestion
    @Test
    @DisplayName("예매 상태는 예매 완료와 예매 취소가 아닌 경우 예외를 발생시킨다.")
    void anotherReservationStatus_Exception() {
        // given & when & then
        assertThatThrownBy(() -> ReservationStatus.valueOf("Proceeding"))
            .isInstanceOf(IllegalArgumentException.class);
    }
```

Reply:
도메인 안에서 별도의 검증 로직을 타는 것이 아닌 valueOf로 처리되는 예외라 말씀하신 것 처럼 필요한 부분만 적어주는 것이 더 간소화되고 보기 좋은 것 같네요! 감사합니다~! 

적용 커밋: [2f7ac96](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/29/commits/2f7ac96907230f47adaab0442f99db601c3ee788)